components/tools/OmeroPy/test/integration/clitest/test_fs.py

```diff
@@ -110,3 +113,33 @@ def testRenameAdminOnly(self, capsys):
             self.cli.invoke(self.args, strict=True)
         out, err = capsys.readouterr()
         assert err.endswith("SecurityViolation: Admins only!\n")
+
+
+class TestFSRoot(RootCLITest):
+
+    def setup_method(self, method):
+        super(TestFSRoot, self).setup_method(method)
+        self.cli.register("fs", FsControl, "TEST")
+        self.args += ["fs"]
+
+    def testFixPyramid(self, capsys):
+        """Test fs fixpyramid"""
+
+        cfg = self.root.sf.getConfigService()
+        pixels_dir = path(cfg.getConfigValue("omero.data.dir")) / "Pixels"
+        iid = self.importSingleImage().id.val
+        subdir = path(long_to_path(iid, root=pixels_dir)).parent
+        subdir.makedirs_p()
+        pyramid_file = subdir / ("%s_pyramid" % iid)
+        pyramid_file.touch()
+        assert pyramid_file.exists()
+        self.args += ["fixpyramid", "Image:%s" % iid]
+        self.cli.invoke(self.args, strict=True)
+        assert not pyramid_file.exists()
```

Comment:
Yes! I was looking at it. The final assert shows that the file is still there:

```
>       assert not pyramid_file.exists()
E       assert not True
E        +  where True = <bound method path.exists of path(u'/repositories/OMERO-DEV-merge-integration/Pixels/Dir-002/2423_pyramid')>()
E        +    where <bound method path.exists of path(u'/repositories/OMERO-DEV-merge-integration/Pixels/Dir-002/2423_pyramid')> = path(u'/repositories/OMERO-DEV-merge-integration/Pixels/Dir-002/2423_pyramid').exists
```

as indeed it is:

```
colin$ omero -C fs fixpyramid Image:2423
Server: [eel.openmicroscopy.org:4064]eel.openmicroscopy.org:24064
Username: [root]
Password:
Created session 0e4fc3ba-a3a2-4329-b2ed-848ead0452fe (root@eel.openmicroscopy.org:24064). Idle timeout: 10 min. Current group: system
$
```
doesn't fail, but,:

```
[cblackburn@ome-c6100-1 ~]$ ls -l /repositories/OMERO-DEV-merge-integration/Pixels/Dir-002/
total 12
-rw-rw-r-- 1 hudson hudson    8 Apr 28 07:49 2057
-rw-rw-r-- 1 hudson hudson    8 Apr 28 07:57 2096
-rw-rw-r-- 1 hudson hudson    8 Apr 28 07:58 2101
-rw-rw-r-- 1 hudson hudson    0 Apr 28 05:15 2423_pyramid
```